packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.styles.ts

```diff
@@ -104,23 +116,44 @@ export const getStyles = (props: IGroupHeaderStyleProps): IGroupHeaderStyles =>
       classNames.check,
       checkExpandResetStyles,
       {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        // paddingTop and marginTop brought from the DetailsRow.styles.ts with explanation below.
+        // Ensure that the check cell covers the top border of the cell.
+        // This ensures the click target does not leave a spot which would
+        // cause other items to be deselected.
+        paddingTop: 1,
+        marginTop: -1,
         opacity: 0,
+        width: '40px',
+        height: finalRowHeight,
         selectors: {
           ':focus': {
             opacity: 1
           }
-        },
-        width: '40px'
+        }
       }
     ],
     expand: [
       classNames.expand,
       checkExpandResetStyles,
       {
-        width: 36,
-        height: rowHeight,
-        color: palette.neutralSecondary,
-        paddingTop: 4
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        fontSize: compact ? FontSizes.mediumPlus : 18, // we don't have 18px in our Fabric type ramps.
```

Comment:
Should we just add that there? Or is there a reason why we should make an exception to the ramp sizes?